client/src/pages/Categories.js

```diff
@@ -1,12 +1,16 @@
 import React from "react";
 import Navbar from "../components/Navbar";
 import Footer from "../components/Footer";
+import CategoryBox from "../components/CategoryBox";
+import "../styles/categories.css"
```

Comment:
I don't think it's necessary to import the styling here again if it's already in `CategoryBox`, but I might be wrong though..

Reply:
That is for styling how multiple boxes will fit in the page.

Reply:
Ah I see my bad 😆